tox.ini

```diff
@@ -96,3 +96,11 @@ skip_install = True
 commands =
     pip install -e .
     black --check {toxinidir}/src {toxinidir}/tests
+
+[testenv:generate_schema]
+setenv = PYTHONWARNDEFAULTENCODING =
+skip_install = True
```

Comment:
Question: Why skip the install and then install anyway below with `pip install -e .`? What's the difference?

Reply:
Mostly because the ones above it do this. Ideally this should build the wheel with PEP 517 & reuse it, but that is a bigger change. (Or use nox, still a little annoyed tox required me to implement `--outfile`)

I can do it with installs, though, that’s how I did it at first before noticing the others in the file do it this way.

Reply:
Oops, didn't realize we did that elsewhere. I don't personally use the tox targets much. In that case, let's just keep this consistent with what's already there.

Reply:
FYI, the pip install is twice as fast as the default way (which can be fixed with settings I could look up, I think). Might have something to do with why it's that way. My original commit has the default way, so I was able to compare when rebasing.